week2/homework/src/index.js

```diff
@@ -1,3 +1,34 @@
 'use strict';
 
-// TODO: Write the homework code in this file
+const commandList = require('./commandsList');
+const theMethods = require('./methods.js');
+
+switch (commandList[0]) {
```

Comment:
Remember that we would like to to display the `help` list also running `node index.js` (no parameters).
Tip: check if `commandList[0]` is empty :)

Reply:
implemented as requested.